cv_6.tex

```diff
@@ -43,7 +45,7 @@
 
 4051 Kensington Drive {\large\textperiodcentered} Concord {\large\textperiodcentered} California\\ % Your mailing address
 {\Large\Letter} keeganseck@gmail.com {\Large\Telefon} (925) 586-8095 \\
-\faLinkedin \space \href{https://www.linkedin.com/pub/evan-steele/83/1b7/b43}{Keegan Seck} \faYoutubePlay \space \href{https://www.youtube.com/channel/UCWcdZhwDnRL\_jbgaRI7tAGA}{Channel}
+\faLinkedin \space \href{https://www.linkedin.com/in/keegan-seck-93812981/}{Keegan Seck} \faYoutubeSquare \space \href{https://www.youtube.com/channel/UCWcdZhwDnRL\_jbgaRI7tAGA}{Channel}
```

Comment:
`faYoutubeSquare` changed sourced from https://www1.ucdenver.edu/docs/librariesprovider29/college-of-engineering-and-applied-science/web-resources/cheatsheet-_-font-awesome.pdf